src/cppcache/include/geode/geode_base.hpp

```diff
@@ -69,32 +69,9 @@
 #define GF_TEMPLATE_EXPORT
 #endif
 
-#if defined(_MSC_VER)
-/* 32 bit Windows only, for now */
-typedef signed char int8;        /**< single byte, character or boolean field */
-typedef unsigned char uint8;     /**< unsigned integer value */
-typedef signed short int16;      /**< signed 16 bit integer (short) */
-typedef unsigned short uint16;   /**< unsigned 16 bit integer (ushort) */
-typedef signed int int32;        /**< signed 32 bit integer */
-typedef unsigned int uint32;     /**< unsigned 32 bit integer */
-typedef signed __int64 int64;    /**< signed 64 bit integer */
-typedef unsigned __int64 uint64; /**< unsigned 64 bit integer */
-
-// typedef int32            intptr_t; /**< a pointer to a 32 bit integer */
-// typedef uint32           uintptr_t; /**< a pointer to an unsigned 32 bit
-// integer */
-
-/* Windows does not have stdint.h */
-typedef int8 int8_t;
-typedef uint8 uint8_t;
-typedef int16 int16_t;
-typedef uint16 uint16_t;
-typedef int32 int32_t;
-typedef uint32 uint32_t;
-typedef int64 int64_t;
-typedef uint64 uint64_t;
-/* end stdint.h */
+#include <cstdint>
```

Comment:
Keeping it here kept refactor simpler.  I suspect reason it was originally here is because we `typedef` own integer fixed types across all platforms and obviously don't want to do that in every file we use these types.  I'm not sure why they chose without `_t`.  I will move this include to each individual file.